crates/bevy_text/src/text.rs

```diff
@@ -13,6 +13,12 @@ pub struct Text {
     pub alignment: TextAlignment,
 }
 
+#[derive(Component, Debug, Default, Clone, Reflect)]
+#[reflect(Component)]
+pub struct BidiCorrectedText {
```

Comment:
Doc strings please; this is public :)

Reply:
Yes - maybe we can somehow make this private - see my other comment. If not, I'll add doc to clarify the correct use.